ui/user_settings.ts

```diff
@@ -396,57 +399,53 @@ let table_data_shortener = function (cutoff_start, keep_length) {
             return d;
         }
 
-        var shortened = d.substr(cutoff_start, cutoff_start + keep_length - 1);
-        return '<span class="ellipsis" title="'+esc(d)+'">'+shortened+'&#8230;</span>';
+        let shortened = d.substr(cutoff_start, cutoff_start + keep_length - 1);
+        return '<span class="ellipsis" title="' + esc(d) + '">' + shortened + '&#8230;</span>';
     };
 };
 
-function format_ethchain_per_account_data(eth_accounts) {
+function format_ethchain_per_account_data(eth_accounts: Array<any>) {
     let data = [];
     for (let account in eth_accounts) {
         if (eth_accounts.hasOwnProperty(account)) {
             let account_data = eth_accounts[account];
-            let eth_amount = parseFloat(account_data['ETH']);
-            eth_amount = eth_amount.toFixed(settings.floating_precision);
-            let total_usd_value = parseFloat(account_data['usd_value']);
-            total_usd_value = total_usd_value.toFixed(settings.floating_precision);
+            let eth_amount_f = parseFloat(account_data['ETH']);
+            let eth_amount = eth_amount_f.toFixed(settings.floating_precision);
+            let total_usd_value_f = parseFloat(account_data['usd_value']);
```

Comment:
why use different variable names here?

Reply:
Because now we have type checking, we can't assign string to number :-)